SSG + Shotgun - Initial Release

ZedekThePlagueDoctor opened the pull request:
• Totally new animations
• New sounds
• SSG now has single and double fire modes

asantoni commented:
Looks great! Do you mind if I tweak the damage a bit on these before accepting it?

asantoni commented:
Also, what's the original source of the art / sounds for the CREDITS file? Can you add that in?

ZedekThePlagueDoctor commented:
Yes, go ahead.

ZedekThePlagueDoctor commented:
SSG fire / open / close / pickup sounds: Tripwire Interactive
SSG insert sound: id Software

Pump Shotty Sounds: 
Voltara ( css.gamebanana.com/sounds/15332 ) - Pump
Deku Scrub ( css.gamebanana.com/sounds/3818 ) - Fire / Insert
EA Digital Illusions ( css.gamebanana.com/sounds/16524 ) - Insert

SSG sprite: Zedek The PD / osjclatchford http://forum.zdoom.org/viewtopic.php?f=37&t=47894&start=45#p833407
SG sprite: TheFunktasm / Zedek The PD

Commit:
SSG + Shotgun - Initial Release

• Totally new animations
• New sounds
• SSG now has single and double fire modes
